contrib/mesos/pkg/scheduler/fcfs.go

```diff
@@ -52,10 +54,11 @@ func NewAllocationStrategy(fitPredicate podtask.FitPredicate, procurement podtas
 
 type fcfsPodScheduler struct {
 	AllocationStrategy
+	nodeStore cache.Store
```

Comment:
again, don't need the whole Store here, just something like a `NodeLookup` func (see related comment in scheduler.go)